addons/web/static/tests/views/kanban_tests.js

```diff
@@ -174,18 +174,18 @@ QUnit.module('Views', {
         });
 
         // check available actions in kanban header's config dropdown
-        assert.ok(kanban.$('.o_kanban_header:first .o_kanban_config .o_column_archive').length,
-                        "should be able to archive the records");
-        assert.ok(kanban.$('.o_kanban_header:first .o_kanban_config .o_column_unarchive').length,
-                        "should be able to restore the records");
+        // assert.ok(kanban.$('.o_kanban_header:first .o_kanban_config .o_column_archive').length,
```

Comment:
Is it really intended to comment assertions here ? 

Or was is temporary to correct the tests afterward ?